Adyen/UI/Form/Items/Phone Number/Phone Extension Picker/FormPhoneExtensionPickerItemView.swift

```diff
@@ -68,7 +68,13 @@ public final class FormPhoneExtensionPickerItemView: FormItemView<FormPhoneExten
     }
     
     internal func setupView() {
-        let stackView = UIStackView(arrangedSubviews: [countryCodeLabel, chevronView, valueLabel])
+        var arrangedSubviews: [UIView] = []
```

Comment:
```suggestion
        let arrangedSubviews: [UIView]
```

Better make it a let for compile-time safety